packages/engine/tests/assets/functions/loadGLTF.test.ts

```diff
@@ -0,0 +1,17 @@
+import assert from "assert"
+
+import {GLTF, GLTFLoader} from "../../../src/assets/loaders/gltf/GLTFLoader"
+import {AssetLoader} from "../../../src/assets/classes/AssetLoader"
+import {Mesh, Material, MeshBasicMaterial, Texture} from "three"
+
+describe("LoadGLTF", () => {
+    describe("LoadLightmaps", () => {
+        it("should load lightmapped gltf", async () => {
+            AssetLoader.load({url:"assets/lightmap_test.glb"}, async (gltf:GLTF) => {

```

Comment:
while this is fine, an integration test is not really what we need, i think we can avoid the bloat of a glb file by just unit testing the `loadLightmaps` function by mocking some GLTF data

Reply:
Mocking up gltf data for this test is extremely complex and I don't feel confident I can mock up data that correctly replicates the state of a gltf passed to loadLightmaps.

loadLightmaps works with both the loaded scene hierarchy in the gltf.scene object and parsed json in the GLTFParser object gltf.parser, along with loading indexed texture dependencies that are created during an import. In order to mock this data up I'll end up needing to write a GLTF file anyways for the texture dependencies to be set up right. Loading a glb file automatically sets up the scene, parser and dependencies to the right state for the test to be done. 

I can certainly make the glb file smaller though.

Reply:
No stress. 😊